models/Nominee.js

```diff
@@ -0,0 +1,38 @@
+import mongoose, { Schema } from "mongoose";
+
+// const { Schema } = mongoose;
+
+export const nomineeSchema = new Schema({
+    // Complex Object Config
+   
+    year_film: {
+        type: Number,
+        required: true,
+    },
+    year_award: {
+        type: Number,
+        required: true,
+    },
+    ceremony: {
+        type: Number,
+        required: true,
+    },
+    category: {
+        type: String,
+        required: true,
+    },
+    nominee: {
+        type: String,
+        required: true,
+    },
+    film: {
+        type: String,
+        required: false,
```

Comment:
I think you can leave this part {required: false} out, since it defaults to false :)